package.json

```diff
@@ -16,15 +16,15 @@
     "node": ">=0.4"
   },
   "dependencies": {
-    "anchor-markdown-header": "^0.5.5",
-    "htmlparser2": "~3.7.1",
-    "markdown-to-ast": "~3.2.3",
-    "minimist": "~1.1.0",
-    "underscore": ">=1.3.3",
-    "update-section": "^0.3.0"
+    "anchor-markdown-header": "~0.5.5",
+    "htmlparser2": "~3.9.2",
+    "markdown-to-ast": "~3.4.0",
+    "minimist": "~1.2.0",
+    "underscore": "~1.3.3",
```

Comment:
what's the latest underscore? We need to make sure we don't downgrade the version here.
Should be `~<whatever latest version is>`.